packages/aws-cdk-lib/aws-ecs-patterns/README.md

```diff
@@ -60,6 +60,8 @@ You can omit `cluster` and `vpc` to let CDK create a new VPC with two AZs and cr
 
 You can customize the health check for your target group; otherwise it defaults to `HTTP` over port `80` hitting path `/`.
 
+You can customize the health check configuration of the container via the `healthCheck` property; otherwise it defaults to the health check configuration from the container.
```

Comment:
Not sure I understand this part `otherwise it defaults to the health check configuration from the container.`. Whats the container in this case?

Reply:
The container would be the one created by the `ALBFargateService`, it's present in the constructor https://github.com/aws/aws-cdk/pull/28797/files#diff-f6a850ef3fda270031b8c90adaf6ad3ad6dec9f776612695f1758b09306b93b0R90

And this is from the docs of the `Container.healtcheck` property, specifying the default value if omitted

> (property) ContainerDefinitionOptions.healthCheck?: ecs.HealthCheck | undefined
The health check command and associated configuration parameters for the container.
*@default*  
> - Health check configuration from container.

Reply:
ah makes sense, I think a link to the docs with properties that can be defined in `healthCheck` would be helpful (i believe its [this](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-ecs-taskdefinition-healthcheck.html))

Reply:
Great idea, maybe even better to link to the CDK specific ones? What is the usual practice?  

https://docs.aws.amazon.com/cdk/api/v2/docs/aws-cdk-lib.aws_ecs.HealthCheck.html

Reply:
ping

Reply:
Yep that'd be great, thanks

Reply:
Done :muscle: